slack_today_i_did/reports.py

```diff
@@ -110,3 +110,61 @@ def as_dict(self):
             'people_to_bother': self.people_to_bother,
             'time_run': (str(self.time_run) if self.time_run is not None else "")  # noqa: E501
         }
+
+
+class Sessions(object):
+    def __init__(self):
+        self.sessions = {}
+
+    def has_running_session(self, person: str) -> bool:
+        if person not in self.sessions:
+            return False
+
+        return self.sessions[person]['is_running']
+
+    def start_session(self, person: str, channel: str) -> None:
+        self.sessions[person] = {
```

Comment:
💭  starting a session twice will wipe your history: maybe less surprise if we don't